### PR TITLE
Fix search to allow spaces

### PR DESCRIPTION
**Pull request by mikix**
Also, show more instructors from search results (~4 instead of ~2).

https://openedx.atlassian.net/browse/DISCO-243

### CODE REVIEW

**Comment by mikix** [comment]
Oh shoot, I didn't add any tests. brb

**Comment by mikix** [comment]
Tests added

File: course_discovery/apps/course_metadata/tests/test_lookups.py [hunk 1]
```diff
@@ -96,27 +96,28 @@ def setUp(self):
...
-        first_instructor = PersonFactory(given_name="First Instructor")
-        second_instructor = PersonFactory(given_name="Second Instructor")
+        first_instructor = PersonFactory(given_name="First", family_name="Instructor")
+        second_instructor = PersonFactory(given_name="Second", family_name="Instructor")
         self.instructors = [first_instructor, second_instructor]
 
         for instructor in self.instructors:
             PositionFactory(organization=self.organizations[0], title="professor", person=instructor)
 
+    def query(self, q):
```
Comment (Dillon-Dumesnil) [inline course_discovery/apps/course_metadata/tests/test_lookups.py:106]: I like this. Nice 👍

**Review by Dillon-Dumesnil — APPROVED** [review]
:shipit:

**Pull request merged**